{"name": "使用有效凭据登录", "status": "failed", "statusDetails": {"message": "AssertionError: 登录失败: None\nassert None == 0\n +  where None = <built-in method get of dict object at 0x1044cff00>('code')\n +    where <built-in method get of dict object at 0x1044cff00> = {'error': '404 Client Error: Not Found for url: http://localhost:8080/user/login'}.get", "trace": "self = <test_demo_api.TestUserAPI object at 0x104106a90>\napi_client = <common.api_client.APIClient object at 0x103f5bd60>\n\n    @allure.story(\"登录接口\")\n    @allure.title(\"使用有效凭据登录\")\n    @pytest.mark.smoke\n    def test_login_with_valid_credentials(self, api_client):\n        \"\"\"测试使用有效凭据登录\"\"\"\n        # 获取测试数据\n        valid_data = login_test_data.get('valid', {})\n    \n        # 发送登录请求\n        with allure.step(\"发送登录请求\"):\n            response = api_client.login(\n                number=valid_data.get('number'),\n                username=valid_data.get('username'),\n                password=valid_data.get('password')\n            )\n    \n        # 断言响应\n        with allure.step(\"验证登录成功\"):\n>           assert response.get('code') == 0, f\"登录失败: {response.get('message')}\"\nE           AssertionError: 登录失败: None\nE           assert None == 0\nE            +  where None = <built-in method get of dict object at 0x1044cff00>('code')\nE            +    where <built-in method get of dict object at 0x1044cff00> = {'error': '404 Client Error: Not Found for url: http://localhost:8080/user/login'}.get\n\ntest_cases/api/test_demo_api.py:36: AssertionError"}, "description": "测试使用有效凭据登录", "steps": [{"name": "发送登录请求", "status": "passed", "start": 1742985538031, "stop": 1742985538055}, {"name": "验证登录成功", "status": "failed", "statusDetails": {"message": "AssertionError: 登录失败: None\nassert None == 0\n +  where None = <built-in method get of dict object at 0x1044cff00>('code')\n +    where <built-in method get of dict object at 0x1044cff00> = {'error': '404 Client Error: Not Found for url: http://localhost:8080/user/login'}.get\n", "trace": "  File \"/Users/Apple/PycharmProjects/himool_auto_case/test_cases/api/test_demo_api.py\", line 36, in test_login_with_valid_credentials\n    assert response.get('code') == 0, f\"登录失败: {response.get('message')}\"\n"}, "start": 1742985538055, "stop": 1742985538055}], "attachments": [{"name": "stderr", "source": "ba23bd85-9ea1-4095-8c9e-63d28f2a160f-attachment.txt", "type": "text/plain"}], "start": 1742985538031, "stop": 1742985538056, "uuid": "75d550f2-af36-475d-bf5c-ce62a2fe9e44", "historyId": "b2c3e5718c1ed6d2532f8cc605d4e167", "testCaseId": "b2c3e5718c1ed6d2532f8cc605d4e167", "fullName": "test_cases.api.test_demo_api.TestUserAPI#test_login_with_valid_credentials", "labels": [{"name": "feature", "value": "用户管理"}, {"name": "story", "value": "登录接口"}, {"name": "epic", "value": "API测试"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "test_cases.api"}, {"name": "suite", "value": "test_demo_api"}, {"name": "subSuite", "value": "TestUserAPI"}, {"name": "host", "value": "MacBook-Pro.local"}, {"name": "thread", "value": "34680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.api.test_demo_api"}]}